python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -36,6 +36,70 @@
 
 import Gaffer
 import GafferScene
+import GafferUI
+
+##########################################################################
+# UI for the state plug that allows setting the state through buttons
+##########################################################################
+
+from collections import namedtuple
+
+ButtonLabels = namedtuple( 'ButtonLabels', 'startButton pauseButton')
+
+class InteractiveRenderingStateWidget( GafferUI.PlugValueWidget ) :
```

Comment:
Prefix with _, to make it clear that this is not part of the public API. Gaffer's codebase doesn't really follow the "we're all consenting adults" python approach of having everything hang out in public - things become unmaintainable really quickly if you don't know what client code might be using. In this case, it's hard to imagine client code accessing this particular class, but it's a principle worth applying everywhere consistently (and it's also sometimes scary what does end up getting used externally).

While we're here, _StatePlugValueWidget would probably be the more convention-following name.